data/language/en-GB.txt

```diff
@@ -890,7 +890,7 @@ STR_1506    :{SMALLFONT}“The litter here is really bad”
 STR_1507    :{SMALLFONT}“I can't find the park exit”
 STR_1508    :{SMALLFONT}“I want to get off {STRINGID}”
 STR_1509    :{SMALLFONT}“I want to get out of {STRINGID}”
-STR_1510    :{SMALLFONT}“I'm not going on {STRINGID} - It isn't safe”
+STR_1510    :{SMALLFONT}“I'm not going on {STRINGID} - it isn't safe”
```

Comment:
In this case I think all peep thoughts should have proper punctuations, with added periods, and proper capitalization.  I know it won't matter most of the time, but in the instances where peeps are saying something excitedly (ie, "I really liked that ride!") they would have punctuation for emphasis and it's more consistent to always have punctuation on thoughts.

As far as the capitalization, I think the same thing applies, since grammar rules are about replicating human thoughts in the most readable fashion.